### ISSUE TITLE
Program Does not work!

### ISSUE THREAD

**Issue by sirhezar** [thelinuxchoice/instashell#64]
Unfortunately, the program cannot find the correct password Now!
Not releasing a new update? That the bug be fixed?

**Comment by amirrezagh75** [comment]
Yeah. I got same issue, it cant find the correct password. I created a fake account and put the password in the list, but it passed the password and at the end it didn't work!

**Comment by sourabhh1** [comment]
same issue

**Comment by ozmoz5566** [comment]
same issue it cant find password
